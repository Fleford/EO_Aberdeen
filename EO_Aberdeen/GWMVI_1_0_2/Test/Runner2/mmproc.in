"..\..\bin\mf2005dbl ..\data\supply2.nam"             # Item 1: Command
..\data\supply2.nam     # ITEM 2: NAMEFILE NAME
supply2.lst     # ITEM 2A: LISTFILE NAME
           1          25          30          12
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
           1
   0.000000000000000                                  # ITEM 3: HNOFLO
                   1                                  # ITEM 4: IFREFM
   0.000000000000000                                  # ITEM 5:  HDRY
  0.1000000000000000E-05                              # ITEM 6: HCLOSE
supply2hd.bhd     # ITEM 7: BINARY FILE FOR HEADS
none     # ITEM 8: BINARY FILE FOR FLOW
                   1                                  # ITEM 9: NABF - Number of other binary files
SFR-FLOW  supply2sf.cbc     # ITEM 10: BINARY FILE FOR SFR-FLOW
           4      # ITEM 11: NMQWEL -- Number of managed WEL cells
supply2.wel     # ITEM 12: WELL FILE NAME
Q1         1         6        22   -8.914906388132324E+003  # Item 13A: Name Lay Row Col dummyQ
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
Q2         1        16        17   -8.328578524251199E+003  # Item 13A: Name Lay Row Col dummyQ
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
Q3         1         8        28   -7.077199483971175E+003  # Item 13A: Name Lay Row Col dummyQ
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
Q4         1        15        19   -1.415062921959331E+004  # Item 13A: Name Lay Row Col dummyQ
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
    1
           0      # ITEM 14: NMQMNW -- Number of managed MNW wells
                  16                                # ITEM 17: NSV - number of simulated values
Q1c         HEAD
         1         6        22         1         # Item 18B: LAY,ROW,COL,SP
Q2c         HEAD
         1        16        17         1         # Item 18B: LAY,ROW,COL,SP
Q3c         HEAD
         1         8        28         1         # Item 18B: LAY,ROW,COL,SP
Q4c         HEAD
         1        15        19         1         # Item 18B: LAY,ROW,COL,SP
S01_9       SFR-FLOW
         1        14         9
S01_10      SFR-FLOW
         1        14        10
S01_11      SFR-FLOW
         1        14        11
S01_12      SFR-FLOW
         1        14        12
S02_11      SFR-FLOW
         1        21        11
S02_12      SFR-FLOW
         1        21        12
S03_9       SFR-FLOW
         2         8         9
S03_10      SFR-FLOW
         2         8        10
S03_11      SFR-FLOW
         2         8        11
S03_12      SFR-FLOW
         2         8        12
S04_11      SFR-FLOW
         3         5        11
S04_12      SFR-FLOW
         3         5        12
..\data\supply2.sfr     # ITEM 19: SFR INPUT FILE
 0.1000E-04             # ITEM 21: EPSQNET
