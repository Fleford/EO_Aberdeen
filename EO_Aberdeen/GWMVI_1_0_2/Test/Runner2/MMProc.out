 Message output for MMProc

COMMAND: ..\..\bin\mf2005dbl ..\data\supply2.nam
NAME FILE: ..\data\supply2.nam
LIST FILE: supply2.lst
NLAY:   1
NROW:     25
NCOL:     30
NPER:     12
NSTP values have been read
HNOFLO:   0.000000    
IFREFM: 1
HDRY:   0.000000    
HCLOSE:  0.1000000E-05
HEADFILE: supply2hd.bhd
FLOWBUDFILE: none
NABF: 1
Binary budget file containing data of type SFR-FLOW:  supply2sf.cbc
NMQWEL:     6
WELFILE: supply2.wel
 #  WEL file specifying no wells.
 #
 #  A file like this is required by GWM-VI
 #  when the original model does not use
 #  the Well Package but the GWM management
 #  model includes flow-rate decision
 #  variables to be simulated with the
 #  Well Package.
 No named parameters
 In Well Package MXACTW=0


     0 Well parameters
NMQMNW:     0


                     DELR =   200.000    

                     DELC =   200.000    

 TOP ELEVATION OF LAYER 1 =   100.000    

   MODEL LAYER BOTTOM EL. =   0.00000     FOR LAYER   1
NSV:     18
 #  Supply2 test problem for GWM-VI
 #  The eighth entry in the first data line is ISTCB2 -
 #    Setting this to a negative number indicates that
 #    streamflows out of the reach will be saved to a
 #    binary file.  The absolute value of ISTCB2 is the
 #    unit to which unformatted flows will be written.
 #
